x11-misc/xflux-gui/xflux-gui-1.1.10_p20180910.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+MY_PV="a3b55da976053fc978b29d191db52dfb8da2f8ee"
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 gnome2-utils
+
+DESCRIPTION="A GUI for f.lux"
+HOMEPAGE="https://justgetflux.com/"
+SRC_URI="https://github.com/xflux-gui/fluxgui/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Please make distfile name match directory name in S.

Reply:
Done.